BinPacker -s <read_type> -p <sequencing_type> -l <left_reads> -r <right_reads> -d -o <output_diretory> -m <library_type>
## where -s is "fq" (fastq) or "fa" (fasta)
## where -p is "pair" or "single"
## where -l is the name and path of the left reads
## where -r is the name and path of the right reads
## where -d removes duplicated transcripts
## where -o is the name and path of the output_directory
## where -m is the library type orientation - for pair, either "RF" or "FR" - for single, either "R" or "F"
